src/core/qgsgeometryvalidator.cpp

```diff
@@ -302,12 +325,12 @@ void QgsGeometryValidator::run()
 
             if ( ringInRing( mp[i][0], mp[j][0] ) )
             {
-              emit errorFound( QgsGeometry::Error( QObject::tr( "polygon %1 inside polygon %2" ).arg( i ).arg( j ) ) );
+              emit errorFound( QgsGeometry::Error( QObject::tr( "Polygon %1 inside polygon %2" ).arg( i ).arg( j ) ) );
```

Comment:
If we're fixing the language here, this should be "Polygon %1 lies inside polygon %2"